lib/experimental/auto-inserting-blocks.php

```diff
@@ -256,3 +323,22 @@ function gutenberg_serialize_block( $block ) {
 function gutenberg_serialize_blocks( $blocks ) {
 	return implode( '', array_map( 'gutenberg_serialize_block', $blocks ) );
 }
+
+/**
+ * Register the `auto_insert` field for the block-types REST API controller.
+ *
+ * @return void
+ */
+function gutenberg_register_auto_insert_rest_field() {
+	register_rest_field(
+		'block-type',
+		'auto_insert',
+		array(
+			'schema' => array(
+				'description' => __( 'Auto Insert.', 'default' ),
```

Comment:
I don't know anything about anything here but I don't feel like this schema description is helping me.

Are we trying to communicate what the API field is or means? When I read the description I think of a boolean, whether to insert or not, but then it's an object so I'm not sure.

Is this a field where I submit a block to auto insert? What is the kind of object it expects?

Is "Auto Insert." here being used as a noun or verb?

Reply:
All very reasonable critique! I'll try to come up with a better description 😊

Reply:
4d29580544. LMK if you'd like us to tweak it some more.